src/api/public/apidocs-new/paths/build_project.yaml

```diff
@@ -0,0 +1,99 @@
+get:
+  summary: Get a simple directory listing of all repositories for the specified project
+  description: Get a simple directory listing of all repositories for the specified project
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
+      name: project
+      schema:
+        type: string
+      required: true
+      description: Name of the project.
+      example: 'home:foo'
+  responses:
+    '200':
+      description: |
+        OK
+
+        XML Schema used for body validation: [directory.xsd](../schema/directory.xsd)
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/directory.yaml'
+          example:
+            entry:
+              - name: 'openSUSE_Tumbleweed'
+              - name: 'openSUSE_Leap_15.3'
```

Comment:
This response 200 returning a list of **repositories** is very common. Does make sense to move it to its own file and add a reference to it whenever it's needed?

Reply:
Makes sense, as discussed lets do it in a batch as a follow up :)